cves/CVE-2008-2364.yml

```diff
@@ -54,7 +58,7 @@ fixes:
    - commit: 52d29a43ce04c396da3ae26c16c04b47a9f77ff7
      note:
    - commit: 4711f90002dea305600274287e470b696f6ae79b
-     note:
+     note: Added checks for the amount of interim respones sent
 vccs:
   - commit: 4785974c5a8a5f485052d2ea8da68ffef2c2846c
     note:
```

Comment:
What was it about this commit that contributed the vulnerability?

Reply:
Nice catch, forgot to add a note here.